docs/cli/combined_markdown.md

```
@@ -0,0 +1,1420 @@
+      "source": "README.md",
+      "target": "docs/quickstart.md",
+      "type": "references",
+      "confidence": 1.0,
+      "timestamp": 1736637890000
+    }
+  ]
+}
+```
+
+DOT for visualization:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)